docs/COMPILE.md

```diff
@@ -25,14 +25,19 @@ sudo systemctl restart systemd-binfmt
 - You can use `make -j1`, `make -j2` with less jobs to prevent running out of memory
 - You can also add `-DBAD_SIGNAL=ON` to the cmake command if you are on a Linux Kernel mixed with Android, like on RK3588.
 
-### What about BOX32
+### Note about Box32
 
-if you want to build Box64 with the Box32 option, you will need to add `-DBOX32=ON` to the cmake command. That will enable 32bits process to be run with Box64.
+If you want to build Box64 with the Box32 option, you will need to add `-DBOX32=ON` to the cmake command. That will enable 32bits process to be run with Box64.

```

Comment:
Can you explain the necessity of option ``BOX32`, which allows 32-bit binaries running on Box64?

Reply:
I'll do that in another commit